src/msw/spinctrl.cpp

```diff
@@ -758,10 +734,24 @@ wxSize wxSpinCtrl::DoGetSizeFromTextSize(int xlen, int ylen) const
     return tsize;
 }
 
+int wxSpinCtrl::GetOverlap() const
+{
+    RECT rectText;
+    ::GetWindowRect(m_hwndBuddy, &rectText);
```

Comment:
I've changed these calls to `wxGetWindowRect()` which discovered the problem with using the invalid `HWND` when this function was called before the window was created, so I had to add a check for this too.